packages/stories/src/Components/Video/Video.component.tsx

```diff
@@ -30,11 +30,12 @@ export function Video(props: IStoryComponentProps) {
     if (!videoRef.current) {
       return;
     }
-    if (isPaused && !videoRef.current.paused) {
-      videoRef.current.pause();
-      return;
-    }
-    videoRef.current.play().catch(() => {
+    
+    videoRef.current.play().then(() => {
```

Comment:
will it not cause a play pause glitch ?

Reply:
No, these changes fix a problem with this issue on the top

Reply:
In the original code happens next error

> DOMException: The play() request was interrupted because video-only background media was paused to save power